src/definitions/modules/checkbox.js

```diff
@@ -829,6 +829,7 @@ $.fn.checkbox.settings = {
   // delegated event context
   uncheckable         : 'auto',
   fireOnInit          : false,
+  enableEnterKey      : true,
```

Comment:
So this change makes a standard (non FUI / html default) checkbox is also triggered by ENTER Key, which is not triggered right now? 

If so, it sounds breaking change.
I suggest to enable this option only for non-standard checkbox, something like this.
```javascript
enableEnterKey : $module.is("input:not([type=checkbox])")
```

Reply:
The non Fui / standard html checkbox is unaffected by this change. Just remove the js code. See https://jsfiddle.net/7kqL485z/

Reply:
Just in case, you are still confused:
> So this change makes a standard (non FUI / html default) checkbox is also triggered by ENTER Key, which is not triggered right now?

No, it's the over way around: html default is to _not_ support ENTER key and this PR makes it possible to also achieve the same behavior for FUI initialized checkboxes as well.
Sorry, if that was not clear enough from the description

Before this change the ENTER key was always supported for FUI initialized checkboxes, that's why the default for `enableEnterKey` is set to `true` to avoid a breaking change.